docstore/drivertest/drivertest.go

```diff
@@ -1433,28 +1430,41 @@ func testUpdateQuery(t *testing.T, coll *ds.Collection) {
 	ctx := context.Background()
 	addQueryDocuments(t, coll)
 
-	err := coll.Query().Where("Player", "=", "fran").Update(ctx, docstore.Mods{"Score": 13, "Time": nil})
-	if err != nil {
-		t.Fatal(err)
-	}
-	got := mustCollectHighScores(ctx, t, coll.Query().Get(ctx))
-	for _, g := range got {
-		g.DocstoreRevision = nil
+	check := func(q *ds.Query, filter func(*HighScore) bool) {
+		t.Helper()
+		err := q.Update(ctx, docstore.Mods{"Score": 13, "Time": nil})
+		if err != nil {
+			t.Fatal(err)
+		}
+		got := mustCollectHighScores(ctx, t, coll.Query().Get(ctx))
+		for _, g := range got {
+			if g.DocstoreRevision == nil {
```

Comment:
You can use `checkHasRevisionField` here.

Reply:
This seems simpler and clearer, since I know the type and field name.